src/utilities/parameters.ts

```diff
@@ -39,16 +40,23 @@ export function getFormParameters(
   const formParameters = Object.entries(parametersMap).map(([name, { order, schema }]) => {
     const arg: Argument = argumentsMap[name];
     const defaultArg: Argument | undefined = defaultArgumentsMap[name];
-    const value = getArgument(arg, schema, defaultArg);
+    const { value, valueSource } = getArgument(arg, schema, defaultArg);
     const required = requiredParameters.indexOf(name) > -1;
 
+    // DOT:
```

Comment:
Can we make the info in this comment a doc comment above `getArgument`?

```ts
/**
 * Determines a value for an argument, which can come from multiple data sources.
 * ...
 */
export function getArgument(
  value: Argument,
  schema: ValueSchema,
  defaultValue?: Argument,
): { value: any; valueSource: ValueSource } {
  ...
}
```

Reply:
Whoops missed this old comment, was just for development purposes so I'll clean that up.